diff --git a/node_modules/@langchain/core/node_modules/ansi-styles/index.js b/node_modules/@langchain/core/node_modules/ansi-styles/index.js
index a9eac58..ced8738 100644
--- a/node_modules/@langchain/core/node_modules/ansi-styles/index.js
+++ b/node_modules/@langchain/core/node_modules/ansi-styles/index.js
@@ -127,12 +127,14 @@ function assembleStyles() {
 		},
 		hexToRgb: {
 			value: hex => {
-				const matches = /(?<colorString>[a-f\d]{6}|[a-f\d]{3})/i.exec(hex.toString(16));
+				// const matches = /(?<colorString>[a-f\d]{6}|[a-f\d]{3})/i.exec(hex.toString(16));
+        const matches = /(?:[a-f\d]{6}|[a-f\d]{3})/i.exec(hex.toString(16));
 				if (!matches) {
 					return [0, 0, 0];
 				}
 
-				let {colorString} = matches.groups;
+				// let {colorString} = matches.groups;
+        let colorString = matches[0];
 
 				if (colorString.length === 3) {
 					colorString = colorString.split('').map(character => character + character).join('');
