diff --git a/node_modules/@langchain/openai/dist/chat_models.js b/node_modules/@langchain/openai/dist/chat_models.js
index dbd4529..3922bf8 100644
--- a/node_modules/@langchain/openai/dist/chat_models.js
+++ b/node_modules/@langchain/openai/dist/chat_models.js
@@ -701,6 +701,8 @@ export class ChatOpenAI extends BaseChatModel {
                 baseURL: endpoint,
                 timeout: this.timeout,
                 maxRetries: 0,
+                stream: false,
+                fetch: fetch,
             };
             if (!params.baseURL) {
                 delete params.baseURL;
