diff --git a/node_modules/@langchain/core/node_modules/camelcase/index.js b/node_modules/@langchain/core/node_modules/camelcase/index.js
index 6ff4ee8..b73cdeb 100644
--- a/node_modules/@langchain/core/node_modules/camelcase/index.js
+++ b/node_modules/@langchain/core/node_modules/camelcase/index.js
@@ -1,10 +1,14 @@
 'use strict';
 
-const UPPERCASE = /[\p{Lu}]/u;
-const LOWERCASE = /[\p{Ll}]/u;
-const LEADING_CAPITAL = /^[\p{Lu}](?![\p{Lu}])/gu;
-const IDENTIFIER = /([\p{Alpha}\p{N}_]|$)/u;
-const SEPARATORS = /[_.\- ]+/;
+// const UPPERCASE = /[\p{Lu}]/u;
+// const LOWERCASE = /[\p{Ll}]/u;
+// const LEADING_CAPITAL = /^[\p{Lu}](?![\p{Lu}])/gu;
+// const IDENTIFIER = /([\p{Alpha}\p{N}_]|$)/u;
+// const SEPARATORS = /[_.\- ]+/;
+const LOWERCASE = /[a-z]/
+const LEADING_CAPITAL = /^[A-Z](?![A-Z])/g
+const IDENTIFIER = /([A-Za-z0-9_]|$)/
+const SEPARATORS = /[_.\- ]+/
 
 const LEADING_SEPARATORS = new RegExp('^' + SEPARATORS.source);
 const SEPARATORS_AND_IDENTIFIER = new RegExp(SEPARATORS.source + IDENTIFIER.source, 'gu');
