diff --git a/node_modules/camelcase/index.js b/node_modules/camelcase/index.js
index 6ff4ee8..47430a7 100644
--- a/node_modules/camelcase/index.js
+++ b/node_modules/camelcase/index.js
@@ -1,9 +1,9 @@
 'use strict';
 
-const UPPERCASE = /[\p{Lu}]/u;
-const LOWERCASE = /[\p{Ll}]/u;
-const LEADING_CAPITAL = /^[\p{Lu}](?![\p{Lu}])/gu;
-const IDENTIFIER = /([\p{Alpha}\p{N}_]|$)/u;
+const UPPERCASE = /[A-Z]/;
+const LOWERCASE = /[a-z]/;
+const LEADING_CAPITAL = /^[A-Z](?![A-Z])/g;
+const IDENTIFIER = /([A-Za-z0-9_]|$)/;
 const SEPARATORS = /[_.\- ]+/;
 
 const LEADING_SEPARATORS = new RegExp('^' + SEPARATORS.source);
