diff --git a/node_modules/langchain/dist/chat_models/openai.js b/node_modules/langchain/dist/chat_models/openai.js
index 63b3cab..9de00f1 100644
--- a/node_modules/langchain/dist/chat_models/openai.js
+++ b/node_modules/langchain/dist/chat_models/openai.js
@@ -632,6 +632,8 @@ export class ChatOpenAI extends BaseChatModel {
                 baseURL: endpoint,
                 timeout: this.timeout,
                 maxRetries: 0,
+                stream: false,
+                fetch: fetch,
             };
             if (!params.baseURL) {
                 delete params.baseURL;
diff --git a/node_modules/langchain/dist/util/@cfworker/json-schema/src/dereference.js b/node_modules/langchain/dist/util/@cfworker/json-schema/src/dereference.js
index 035e3fe..a36fc68 100644
--- a/node_modules/langchain/dist/util/@cfworker/json-schema/src/dereference.js
+++ b/node_modules/langchain/dist/util/@cfworker/json-schema/src/dereference.js
@@ -63,8 +63,8 @@ export let initialBaseURI =
 // @ts-ignore
 typeof self !== "undefined" && self.location
     ? //@ts-ignore
-        /* #__PURE__ */ new URL(self.location.origin + self.location.pathname + location.search)
-    : /* #__PURE__ */ new URL("https://github.com/cfworker");
+        /* #__PURE__ */ { href: self.location.origin + self.location.pathname + location.search }
+    : /* #__PURE__ */ { href: "https://github.com/cfworker" };
 export function dereference(schema, lookup = Object.create(null), baseURI = initialBaseURI, basePointer = "") {
     if (schema && typeof schema === "object" && !Array.isArray(schema)) {
         const id = schema.$id || schema.id;
diff --git a/node_modules/langchain/dist/util/load.js b/node_modules/langchain/dist/util/load.js
index e993007..5b44908 100644
--- a/node_modules/langchain/dist/util/load.js
+++ b/node_modules/langchain/dist/util/load.js
@@ -1,6 +1,6 @@
 export const loadFromFile = async (uri, loader, values = {}) => {
     try {
-        const fs = await import("node:fs/promises");
+        // const fs = await import("node:fs/promises");
         return loader(await fs.readFile(uri, { encoding: "utf-8" }), uri, values);
     }
     catch (e) {
diff --git a/node_modules/langchain/dist/util/stream.js b/node_modules/langchain/dist/util/stream.js
index 51e65e3..12a75b7 100644
--- a/node_modules/langchain/dist/util/stream.js
+++ b/node_modules/langchain/dist/util/stream.js
@@ -1,3 +1,4 @@
+class ReadableStream { }
 /*
  * Support async iterator syntax for ReadableStreams in all environments.
  * Source: https://github.com/MattiasBuelens/web-streams-polyfill/pull/122#issuecomment-1627354490
