diff --git a/node_modules/node-domexception/index.js b/node_modules/node-domexception/index.js
index b7bbe95..b043f8e 100644
--- a/node_modules/node-domexception/index.js
+++ b/node_modules/node-domexception/index.js
@@ -1,6 +1,6 @@
 /*! node-domexception. MIT License. Jimmy Wärting <https://jimmy.warting.se/opensource> */
 
-if (!globalThis.DOMException) {
+if (!_globalThis.DOMException) {
   try {
     const { MessageChannel } = require('worker_threads'),
     port = new MessageChannel().port1,
@@ -8,9 +8,9 @@ if (!globalThis.DOMException) {
     port.postMessage(ab, [ab, ab])
   } catch (err) {
     err.constructor.name === 'DOMException' && (
-      globalThis.DOMException = err.constructor
+      _globalThis.DOMException = err.constructor
     )
   }
 }
 
-module.exports = globalThis.DOMException
+module.exports = _globalThis.DOMException
