diff --git a/node_modules/serialize-error/error-constructors.js b/node_modules/serialize-error/error-constructors.js
index bdad603..5eba6c6 100644
--- a/node_modules/serialize-error/error-constructors.js
+++ b/node_modules/serialize-error/error-constructors.js
@@ -8,12 +8,12 @@ const list = [
 	URIError,
 
 	// Built-in errors
-	globalThis.DOMException,
+	_globalThis.DOMException,
 
 	// Node-specific errors
 	// https://nodejs.org/api/errors.html
-	globalThis.AssertionError,
-	globalThis.SystemError,
+	_globalThis.AssertionError,
+	_globalThis.SystemError,
 ]
 	// Non-native Errors are used with `globalThis` because they might be missing. This filter drops them when undefined.
 	.filter(Boolean)
